tests/xbee_packet.py

```diff
@@ -104,6 +122,26 @@ def test_unserialize(self):
         })
         self.assertFalse(self.packet._private)
 
+    def test_unserialize_object_pack(self):
```

Comment:
This should be `test_unserialize_object_packed(self)` (pack -> packed) to be in line with the serialization test name.